readthedocs/support/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from readthedocs.support.views import FrontWebhook
+
+urlpatterns = [
+    url(r'^front-webhook/$', FrontWebhook.as_view(), name='front_webhook'),
```

Comment:
I mentioned in -ops that we could call all of this `frontapp` to make it clearer that we are refering the Front Support service.